docs/content/9.deploy/2.from-proj.md

```diff
@@ -21,10 +21,13 @@ navigation:
 | - | - | - |
 | `--access-key` | 字符串 | 火山引擎AK |
 | `--secret-key` | 字符串 | 火山引擎SK |
-| `--vefaas-app-name` | 字符串 | 火山引擎 VeFaaS 平台应用名称 |
-| `--veapig-instance-name` | 字符串 | 火山引擎 APIG 实例名称 |
-| `--veapig-service-name` | 字符串 | 火山引擎 APIG 服务名称 |
-| `--veapig-upstream-name` | 字符串 | 火山引擎 APIG Upstream 名称 |
+| `--vefaas-app-name` | 字符串 | 火山引擎函数服务应用名称 |
```

Comment:
why we need the app name for newly deployed app?

Reply:
This parameter existed already. I just changed its description.